### ISSUE TITLE
change alarm type greyed out

### ISSUE THREAD

**Issue by algazarra** [andrefbsantos/boilr#104]
![screenshot_2015-03-13-00-32-34](https://cloud.githubusercontent.com/assets/7598320/6631090/b59430b8-c918-11e4-927f-c423e51d9c9c.png)

after creating an alarm, if we go to the alarm's settings, I'm not able to change the alarm type.

**Comment by dllud** [comment]
Sure that's on purpose. Besides being technically difficult to change alarm type once created, it also would be a strange and little used use case. You would only keep Exchange and Pair and have to set all the rest.

PS. You agreed to this on the beginning.

**Comment by algazarra** [comment]
yes I remember, but I don't remember why it had to behave like that. Was there a technical reason?

**Comment by dllud** [comment]
As I said during that discussion it is doable but too much work for little use. Changing alarm type requires deletion of current alarm, creation of a new one and replacement of the DB (preferably using the same ID).

We might implement this one day if people start coming here by the dozen and requesting this.

**Comment by algazarra** [comment]
you mean, too much work coding?
I'd leave this open. having the option greyed out doesn't look very good and isn't very logical since you can change the alarm type multiple times when creating it. if you can change other parameters, why would this be the only one greyed out, you know?
